web/modules/custom/os2forms_selvbetjening/src/Helper/FormHelper.php

```diff
@@ -35,6 +37,20 @@ public function formAlter(array &$form, FormStateInterface $form_state, string $
     if ('webform_add_form' === $form_id) {
       $form['category']['#description'] = $webform_category_description;
     }
+
+    if ('maestro_interactive_form' === $form_id) {
+      if (isset($form['error']['#markup'])) {
```

Comment:
Consider combining to 

```suggestion
    if ('maestro_interactive_form' === $form_id && isset($form['error']['#markup'])) {
```

Reply:
Combined.